spec/spec_helper.rb

```diff
@@ -0,0 +1,14 @@
+require 'puppetlabs_spec_helper/module_spec_helper'
+
+RSpec.configure do |config|
+  config.hiera_config = 'spec/fixtures/hiera/hiera.yaml'
```

Comment:
I don't see the `fixtures` folder

Reply:
Copied spec_helper.rb from different module. Will remove :)